plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/glue/TestHiveGlueMetastore.java

```diff
@@ -152,34 +158,22 @@ public void testRenameTable()
         // rename table is not yet supported by Glue
     }
 
-    @Override
-    public void testPartitionStatisticsSampling()
-    {
-        // Glue metastore does not support column level statistics
-    }
-
-    @Override
-    public void testUpdateTableColumnStatistics()
-    {
-        // column statistics are not supported by Glue
-    }
-
     @Override
     public void testUpdateTableColumnStatisticsEmptyOptionalFields()
+            throws Exception
     {
-        // column statistics are not supported by Glue
-    }
-
-    @Override
-    public void testUpdatePartitionColumnStatistics()
-    {
-        // column statistics are not supported by Glue
+        assertThatThrownBy(super::testUpdateTableColumnStatisticsEmptyOptionalFields)
```

Comment:
Let's add some comment about why this is failing? AFAIR you said that even though Glue API docs say that some fields are "not required" not setting them when making the API call causes an error. Correct?

Reply:
I added a comment in the discussion about this issue